apiary.apib

```diff
@@ -1738,6 +1738,699 @@ root will not be listable, as it is virtual. Access known paths directly.
 ## restart
 
 # Group update
+Update IPFS
+   
+## update [GET /update]
+
++ Request Without Arguments
+
+    #### curl
+
+        curl -i "http://localhost:5001/api/v0/update"
+
+    + Body
+
+        ```
+        curl -i "http://localhost:5001/api/v0/update"
+        ```
+
++ Response 200
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Output, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Output, X-Chunked-Output
+        Content-Type: text/plain
+        Server: go-ipfs/0.4.0-dev
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        X-Stream-Output: 1
+        Date: Thu, 11 Feb 2016 22:20:00 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (string)
+
+    + Body
+
+        ```
+        NAME:
+           ipfs-update - update ipfs
+
+        USAGE:
+           ipfs-update [global options] command [command options] [arguments...]
+           
+        VERSION:
+           0.1.0
+           
+        COMMANDS:
+           versions print out all available versions
+           version  print out currently installed version
+           install  install a version of ipfs
+           stash    stashes copy of currently installed ipfs binary
+           revert   revert to previously installed version of ipfs
+           fetch    fetch a given (default: latest) version of ipfs
+           help, h  Shows a list of commands or help for one command
+           
+        GLOBAL OPTIONS:
+           --verbose        print verbose output
+           --help, -h       show help
+           --version, -v    print the version
+        ```
```

Comment:
Not sure, we could change go-ipfs to print a different version when called from the api, which is is not cli dependent maybe?